05_stringer/alextmz/main_test.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_iPAddrString(t *testing.T) {
```

Comment:
As mentioned in my last review, you should not use underscores in symbols in Go: https://talks.golang.org/2014/names.slide#9. In this case, I would name the function `TestIPAddrString` - I know this is not an exact match for the case of the type, but `TestipAddrString` is a bit weird, so I'd just go with the former.

Reply:
Agreed, fixed.